ert_shared/ensemble_evaluator/ensemble/prefect.py

```diff
@@ -57,14 +57,15 @@ async def _eq_submit_job(self, script_filename):
 
 
 def _get_executor(custom_port_range, name="local"):
-    _, port, _ = find_available_port(custom_range=custom_port_range, reuse_addr=True)
     if name == "local":
         cluster_kwargs = {
             "silence_logs": "debug",
-            "scheduler_options": {"port": port},
         }
         return LocalDaskExecutor(**cluster_kwargs)
     elif name == "lsf":
+        _, port, _ = find_available_port(
```

Comment:
Is this needed @sondreso ? From what i remember when debugging the port issues - the executor running lsf and pbs should not need a port? And - available port on the node running this code is not equivalent to an available port on a node running on the cluster?  Or am i just thinking and remembering wrong here?